lib/api/readable.js

```diff
@@ -165,7 +169,7 @@ class BodyReadable extends Readable {
     }
 
     return await new Promise((resolve, reject) => {
-      if (this[kContentLength] > limit) {
+      if (this[kContentLength] > limit || this[kBytesRead] > limit) {
```

Comment:
Shall we consider this a breaking change?

Reply:
I don't think so.